### PR TITLE
Add setup-osx setup-unix setup-win32

### PR DESCRIPTION
**Pull request by Lzw104522773**
Resolve the issue of garbled characters when installing with the latest version of weidu.exe.

### CODE REVIEW

File: ntotsc/ntotsc.tp2 [hunk 1]
```diff
@@ -100,7 +100,7 @@ ALWAYS
 				"schinese" => "CP936"
 			END
 			ACTION_DEFINE_ARRAY charsetsReloadArray BEGIN setup journal END
-//			ACTION_DEFINE_ARRAY charsetsNoConvertArray BEGIN ## END
+//			ACTION_DEFINE_ARRAY charsetsNoConvertArray BEGIN setup-osx setup-unix setup-win32 END
```
Comment (Roberciiik) [inline ntotsc/ntotsc.tp2:103]: This line is commented out. Have you tested your solution?

File: ntotsc/ntotsc.tp2 [hunk 2]
```diff
@@ -210,7 +210,8 @@ LANGUAGE
 LANGUAGE
 	~Simplified Chinese(by Archenemy,Latest update by You Ye & Lewis Liu (V5.0.0))~
 	~schinese~
-	~ntotsc/language/schinese/setup.tra~
+        ~ntotsc/language/schinese/setup-%WEIDU_OS%.tra~
+	//~ntotsc/language/schinese/setup.tra~
```
Comment (Roberciiik) [inline ntotsc/ntotsc.tp2:214]: The file `setup.tra` has to be loaded as well to set all items/scripts translations for installer. Please load setup first, and then load `setup-%WEIDU_OS%.tra`.

File: ntotsc/ntotsc.tp2 (hunk 1 above)
Comment (Lzw104522773) [reply ntotsc/ntotsc.tp2]: yes, it need to delete //.

File: ntotsc/ntotsc.tp2 (hunk 2 above)
Comment (Lzw104522773) [reply ntotsc/ntotsc.tp2]: yes,I have done the modify.Thank you.

**Comment by 11jo** [comment]
Setup-win32 is in UTF8 and in, chinese...

**Comment by Roberciiik** [comment]
> Setup-win32 is encoded UTF8 and in Chinese (with symbol)...

What do you mean by "with symbol"?

**Comment by 11jo** [comment]
It's why we talked about last time, for win32 system player use weidu legacy.

weidu legacy don't display symbol (or chinese langage) and accent.

So setup-win32 need to be in english or another language without accents or symbol

**Comment by Roberciiik** [comment]
Yes, English would be the best. But as you can see, Chinese players prioritize EE and new WeiDU over legacy support. These files could be renamed to "install.tra" without specifying OS at all.

**Comment by 11jo** [comment]
Yes, it's what I meant !

Because it's the same as DarkHorizons, it's not needed to add all of this if it's not for make it work on win32 legacy

**Comment by Roberciiik** [comment]
@Lzw104522773 For the correctness sake, could you please publish your changes again to include only single file with commands prompts? In fact, I would suggest doing it this way every time when there is no other setup-WEIDU_OS.tra files in the mod project already.

**Comment by Lzw104522773** [comment]
OK。I‘ll use only one file.

**Pull request merged**